src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -76,6 +76,25 @@ RSVP.on('error', function(reason) {
 
 Please consult your promise library documentation on how to hook into its global unhandled rejection handler, if it exposes one.
 
+&nbsp;
+## Releases
+A release is a version of your code that you deploy to an environment. When you give Sentry information about your releases, you unlock many new features:
+ - Determine the issue and regressions introduced in a new release
+ - Predict which commit caused an issue and who is likely responsible
+ - Resolve issues by including the issue number in your commit message
+ - Receive email notifications when your code gets deployed
+
+Additionally, the Sentry SDK uses releases for applying [source maps]({%- link _documentation/platforms/javascript/sourcemaps/index.md -%}).
+
+&nbsp;
+Setting up releases is a 3-step process:
+1. [Configure Your SDK]({%- link _documentation/workflow/releases.md -%}#configure-sdk)
```

Comment:
This is weird in the JAvaSCript doc -- you've already configured your SDK by this point, right?

Reply:
Yup! Good point. Fixed it!

Reply:
@eurica